var/spack/repos/builtin/packages/openblas/package.py

```diff
@@ -274,6 +275,9 @@ def make_defs(self):
         if self.spec.satisfies('%gcc@:4.8.4'):
             make_defs.append('NO_AVX2=1')
 
+        if self.spec.satisfies('@0.3.13:'):
+            make_defs.append('RANLIB=ranlib')
```

Comment:
Out of curiosity, why was this necessary? Maybe a comment with the explanation would be useful?

Reply:
Sorry, totally forgot to add some comment about this. Should be fixed now.

Reply:
Reading the comment. Was it advertized as a feature or a bug upstream? :slightly_smiling_face: